src/js/components/handles/index.js

```diff
@@ -49,8 +50,8 @@ export default class Handles {
 
         function changeDimensions (evt) {
             move(convertGlobalToLocal(evt, scope.$$parent.getBoundingClientRect()), scope.meta.dimensions);
-
-            scope.$$parent.dispatchEvent(new CustomEvent('source:dimensions'));
+            const toggle_event = createEvent('source:dimensions');
```

Comment:
You can still keep this a one liner, I think `scope.$$parent.dispatchEvent(createEvent('source:dimensions'))` is clear enough (also check other usage)